.github/CONTRIBUTING.md

```diff
@@ -4,13 +4,13 @@ Thank you for wanting to contribute!
 
 Before implementing anything major:
 
-- For new components, [open a GitHub discussion](https://github.com/getlift/lift/discussions/categories/components) and explain the use case with an example
-- For new features, [open a GitHub issue](https://github.com/getlift/lift/issues/new) and explain the use case with an example
+-   For new components, [open a GitHub discussion](https://github.com/getlift/lift/discussions/categories/components) and explain the use case with an example
```

Comment:
Could you exclude documentation (Markdown files) from linting and revert these changes?

I feel like changes in these files aren't ideal:

- the `-    ` indentation isn't natural when writing Markdown
- code examples should be carefully crafted for simplicity/readability and shouldn't necessarily follow the same code formatting as the actual code of the project

I don't think it's necessary to put too many rules on documentation.

Reply:
Done !